src/common.h

```diff
@@ -50,10 +56,11 @@
 #define NAMELEN			0x200
 #define BUFSZ			0x10000
 #define PAYLOADSZ		0x80000 /* 512K: this is our max read limit for libcurl */
-#define WRITETHROUGH		0
-#define WRITEAROUND		1
-#define WRITEBACK		2
+//#define WRITETHROUGH		0
```

Comment:
We can remove these commented out lines.

Reply:
> We can remove these commented out lines.

Fixed in forthcoming commit, FYI these values are currently set at lines:

https://github.com/pkoutoupis/rapiddisk/blob/db51403ad0934462e8cd710da12bd56879808596/src/common.h#L176-L180